apps/docs/content/components/forms/RadioGroup.mdx

```diff
@@ -34,7 +34,7 @@ links:
 
 A `RadioGroup` uses the following components.
 
-<ComposedComponents components={["ErrorMessage", "HelperMessage", "Icon", "IconList", "Label", "Text"]}/>
+<ComposedComponents components={["Icon", "IconList", "Text"]}/>
```

Comment:
It still uses it behind the scene no? shouldn't we still list it? Or since the end user can't do much with this information it's of no use?

Reply:
we only list the components that users can add. They don't care what's behind the scene :P